index.js

```diff
@@ -0,0 +1,15 @@
+const menu = document.querySelector('.menu-button');
+const popUp = document.querySelector('.pop');
+const scrollb = document.querySelector('body');
+
+menu.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  popUp.classList.toggle('active');
+  scrollb.classList.toggle('active');
+});
+
+document.querySelectorAll('.menu-links').forEach((i) => i.addEventListener('click', () => {
+  menu.classList.remove('active');
+  popUp.classList.remove('active');
+  scrollb.classList.remove('active');
+}));
```

Comment:
[OPTIONAL] Please I suggest you refactor this code to a function called `toggleMenu` that toggles the `active` class on the `menu-button`, `pop`, and `body` elements. This function is attached to the click event listener on the `menu-button` element.

The click event listener is simplified on the `menu-links` elements by invoking the `toggleMenu` function directly when any `menu-links` element is clicked.

By doing this, the need to repeat the same code has been eliminated for adding and removing the `active` class on multiple elements. This makes the code more readable, easier to maintain and reduces the chance of errors.

```
function toggleMenu() {
  menu.classList.toggle('active');
  popUp.classList.toggle('active');
  scrollb.classList.toggle('active');
}

menu.addEventListener('click', toggleMenu);

document.querySelectorAll('.menu-links').forEach((i) => i.addEventListener('click', () => {
  toggleMenu();
}));
```